mavlink/modules/flight_input_device.py

```diff
@@ -0,0 +1,34 @@
+from dronekit import connect 
+
+class FlightInputDevice: 
+
+    def __init__(self, address: "str"):
+        """
+        address: tcp address (e.g. "tcp:127.0.0.1:14550")
```

Comment:
yea also say `connection address: TCP address (e.g. "tcp:127.0.0.1:14550")`

Reply:
No, format is `variable_name: Description of what variable_name represents/does` .